.github/workflows/cypress.yaml

```
@@ -81,58 +77,23 @@
-          SPLIT_INDEX: ${{ strategy.job-index }}
+      - name: Fix Permissions for Cypress Test Files
+        run: chmod -R +rwx cypress/e2e  
 
-      - name: Upload cypress screenshots on failure 📸
-        uses: actions/upload-artifact@v3
-        if: failure()
-        with:
-          name: cypress-screenshots
-          path: cypress/screenshots
+      - name: Run Cypress Tests in Parallel
+        run: |
```

actionlint (1.7.3)

shellcheck reported issue in this script: SC2086:info:3:17: Double quote to prevent globbing and word splitting

(shellcheck)